traits/05-maricopa-field-scanner.Rmd

```diff
@@ -1,11 +1,7 @@
----
-title: "Plot level data from the field scanner in Maricopa, AZ"
-author: "David LeBauer, Chris Black"
-date: "`r Sys.Date()`"
-output: md_document
----
+# Plot level data from the field scanner in Maricopa, AZ
+
 ```{r traits-05-mac-traits-setup, include=FALSE}
-knitr::opts_chunk$set(echo = FALSE, cache = TRUE)
+knitr::opts_chunk$set(echo = FALSE, cache = FALSE)
```

Comment:
What are the pros / cons of setting cache=TRUE by default? It seems it would make building the site faster, but does it complicate debugging? What are the disadvantages?

Reply:
I think that setting cache = TRUE by default does make building the book a lot faster. However, I did encounter issues with running code that called cached data. Often, the correct data wasn't there, so columns would be missing.